### DISCUSSION TITLE
In-line issue suppression within Infrastructure as Code material

### DISCUSSION THREAD

**Discussion by huornlmj** [aquasecurity/trivy#4293] [Q&A]
As a user I would like to be able to place checker suppression within my IaC code so that I can selectively accept issues.  Most security tools allow for a number of suppression methods,  e.g. exclude a file, disable a checker by ID, and finally put an inline comment with something like #nosec to skip the line.  Trivy doesn't appear to have this.  In fact it only seems to allow for complete file exclusion which is too far the other direction.

**Comment by knqyf263** [comment]
`#trivy:ignore:` should work. Are you talking about something else?
https://github.com/aquasecurity/trivy/issues/2961

**Comment by itaysk** [comment]
BTW which language you need it for?

**Comment by huornlmj** [comment]
@itaysk K8s YAML object files, but pretty any IaC material Trivy covers.

**Comment by creste** [comment]
I also need `#trivy:ignore:...` to work with IaC, specifically k8s yaml files used as templates in helm charts.
